plugin.video.srgssr_ch_replay/resources/language/resource.language.de_de/strings.po

```diff
@@ -29,24 +29,52 @@ msgid "# (numbers & special characters)"
 msgstr "# (Nummern & Sonderzeichen)"
 
 msgctxt "#30020"
-msgid "Older episodes (page {0} of {1})"
-msgstr "Ältere Ausstrahlungen (Seite {0} von {1})"
+msgid "Page {0}{1} >> more results"
```

Comment:
I like the change :+1:

Reply:
I checked the API, and the latest videos by topic endpoint doesn't return the number of episodes

Reply:
> I checked the API, and the latest videos by topic endpoint doesn't return the number of episodes

Thank you. I'll close that issue...